src/components/Home.css

```diff
@@ -0,0 +1,89 @@
+body {
+  background-image: url('../features/weatherimage.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-attachment: fixed;
+  backdrop-filter: blur(5px);
+}
+
+* {
+  font-family: 'Open Sans', sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
```

Comment:
- [ ] Good job in creating a nice app, but we need to stick to the project design and guide, in the project design, the cards are stacked in a two-column grid separated by the different colour of alternate cards and no spaces between each card while your design is stacked in a one-column grid and there are spaces between cards, kindly fix this
your design :point_down: 
![Screenshot from 2023-09-08 00-47-59](https://github.com/Bupilipili/City-Weather/assets/66097371/edb1f69b-13de-4147-991f-773d768df139)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/Bupilipili/City-Weather/assets/66097371/c32c0d75-c7f7-4b5d-a5b2-d48315842b75)